skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -552,3 +553,21 @@ void MpActor::DropItem(const uint32_t baseId, const Inventory::Entry& entry)
   RemoveItems({ entry });
   // TODO(#1141): reimplement spawning items
 }
+
+void MpActor::SetIsBlockActive(bool active)
+{
+  isBlockActive = active;
+}
+
+bool MpActor::IsBlockActive() const
+{
+  return isBlockActive;
+}
+
+NiPoint3 MpActor::GetViewDirection() const
+{
+  static const float kPi = std::acos(-1.f);
+  static const float kAngleToRadians = kPi / 180.f;
```

Comment:
Can be moved out of this function.
```suggestion
const float kAngleToRadians = std::acos(-1.f) / 180.f;

NiPoint3 MpActor::GetViewDirection() const
{
```